custom/samveg/case_importer/validators.py

```diff
@@ -186,8 +186,8 @@ def run(cls, row_num, raw_row, fields_to_update, import_context, domain_name):
         return fields_to_update, error_messages
 
     @classmethod
-    def _upload_limit_reached(cls, import_context, owner_name, call_number, domain_name):
-        row_limit = Domain.get_by_name(domain_name).operator_call_limit
+    def _upload_limit_reached(cls, import_context, owner_name, call_number, domain):
+        row_limit = OperatorCallLimitSettings.objects.get_or_create(domain=domain).call_limit
```

Comment:
This is called on every row, right? If so, it might be worth making a cached dbaccessor for this value.

Reply:
Had thought about this earlier but dropped the idea since were 3 entries in the table for it. Also, when will cache get refreshed if a new domain entry is added to the table?